db/dataset.py

```diff
@@ -315,3 +315,54 @@ def _delete_snapshots_for_dataset(connection, dataset_id):
         DELETE FROM dataset_snapshot
               WHERE dataset_id = :dataset_id""")
     connection.execute(query, {"dataset_id": dataset_id})
+
+
+def add_recordings(dict, dataset_id):
```

Comment:
This is because it's the input as given to the API endpoint. However, I agree. `(dataset_id, class_name, recordings)` is a good method signature